### PR TITLE
Tableau Pt 2

### PR DESCRIPTION
**Pull request by atatmaja**
No changes made since the last PR, just resubmitting bc some merges got f'd. See previous PR for description.

### CODE REVIEW

File: src/components/shared/TableauEmbed.tsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+import { useMediaQuery } from "react-responsive";
+import { mobileDeviceOrTabletWidth } from "../../constants";
+
+interface TableauEmbedProps {
+    url: string,
+    options?: Record<string, string>
+}
+
+export default function TableauEmbed(props: TableauEmbedProps) {
+    const isMobileDeviceOrTablet = useMediaQuery({ maxDeviceWidth: mobileDeviceOrTabletWidth })
```
Comment (brett-stephen) [inline src/components/shared/TableauEmbed.tsx:15]: are we using this media query?

File: src/components/shared/references/StudiesTable.tsx [hunk 2]
```diff
@@ -2,13 +2,13 @@ import _ from "lodash";
...
-import StudyDetailsModal from "../charts/StudyDetailsModal";
-import ReferenceSearch from '../charts/ReferencesSearch';
-import { sendAnalyticsEvent } from '../../utils/analyticsUtils';
+import { mobileDeviceWidth } from "../../../constants";
+import { AirtableRecord } from "../../../types";
+import Translate from "../../../utils/translate/translateService";
+import { getGeography, getPossibleNullDateString } from "../../../utils/utils";
+import StudyDetailsModal from "./StudyDetailsModal";
+import ReferenceSearch from './ReferencesSearch';
+import { sendAnalyticsEvent } from '../../../utils/analyticsUtils';
```
Comment (brett-stephen) [inline src/components/shared/references/StudiesTable.tsx:11]: Not totally necessary, but we could make imports absolute paths. It makes it cleaner, especially if they are in the source directory ie: `import { mobileDeviceWidth } from "constants";`

**Review by brett-stephen — APPROVED** [review]
2 small comments, but other than that - LGTM

**Review by annoyingpr — COMMENTED** [review]
Leaving this comment from the previous PR. 

image

Would also be nice if these filters for the Tableau embed could show at the top on mobile instead of on the right.

**Review by annoyingpr — APPROVED** [review]
🚀  LGTM once the mobile fixes are wrapped up!

**Comment by atatmaja** [comment]
> Leaving this comment from the previous PR.
> 
> image
> 
> Would also be nice if these filters for the Tableau embed could show at the top on mobile instead of on the right.

@jordanvanwyk flagging this

**Pull request merged**